src/main/java/com/clevertap/apns/Notification.java

```diff
@@ -45,20 +45,29 @@
     private final String token;
     private final String topic;
     private final String collapseId;
+    private final Long expiration;
+    private final Integer priority;
+    private final String uuid;
```

Comment:
Can you change this to a UUID object instead? The reasoning behind it is to ensure that only a UUID value is passed to the Apple gateway, and nothing else.